code/modules/surgery/organs/augments_arms.dm

```diff
@@ -63,7 +63,7 @@
 	if(. & EMP_PROTECT_SELF)
 		return
 	if(prob(15/severity) && owner)
-		to_chat(owner, "<span class='warning'>[src] is hit by EMP!</span>")
+		to_chat(owner, "<span class='warning'>The electromagnetic pulse causes [src] to malfunction!</span>")
```

Comment:
```suggestion
		to_chat(owner, "<span class='warning'>An electromagnetic pulse causes [src] to malfunction!</span>")
```
😏

Reply:
I had considered using "An" instead of "The", but since EMPs are usually very flashy and aren't too common, I figured that "The" would be more appropriate.

Like, if you get hit by an explosion, the message should be "You get knocked back by THE explosion!", not "You get knocked back by AN explosion!".

Reply:
sometimes they aren't flashy tho, like EMP flashlights or stationwide EMP waves
tbh explosion message should probably be "an explosion" as well

Reply:
> sometimes they aren't flashy tho, like EMP flashlights or if EMP blasts from fusion bombs going of is that even a thing it should be
> 
> tbh explosion message should probably be "an explosion" as well

Yeah, but you usually aren't getting hit by two EMPs or two explosions simultaneously.

Reply:
> he's never been spam clicked by a guy with an ion gun

noob